pom.xml

```diff
@@ -8,7 +8,7 @@
     </parent>
 
     <artifactId>read-only-configurations</artifactId>
-    <version>1.11-SNAPSHOT</version>
+    <version>1.12-SNAPSHOT</version>
```

Comment:
Why changing the next version

Reply:
is it not a standard to bump the version of the plugin for an update?

Reply:
That is better left for maven release plugin to do. Last version was 1.10 so the next one should be 1.11 unless we have a reason to skip one. The old value was correct to achieve that

Reply:
Thanks for the explanation